tendermint/src/validator.rs

```diff
@@ -270,18 +277,18 @@ mod tests {
             158_095_448_483_785_107,
         );
         let v3 = make_validator(
-            "EB6B732C4BD86B5FA3F3BC3DB688DA0ED182A7411F81C2D405506B298FC19E52",
+            "76A2B3F5CBB567F0D689D9DF7155FC89A4C878F040D7A5BB85FF68B74D253FC7",
             770_561_664_770_006_272,
         );
```

Comment:
This is *another* example of an Ed(wards)25519 point which failed to validate. It looks like it might've been munged slightly from the similar looking point below (L284).

It looks like these vectors were tested against the Go implementation, so they need to be rechecked for compatibility.

Reply:
I'll recheck them during the weekend.

Reply:
I suspect this is a similar problem as above and the key was simply not valid. On the other hand I'm surprised that the other validator keys here (v1, v2) are valid 🤔 As we probably generated them in the same way. From looking at the `decompress` code in the rust lib I don't see any discrepancies though. 

I would assume that both libraries are properly tested against many test-vectors? https://ed25519.cr.yp.to/python/sign.input (true for the go lib not sure about the rust impl)

A simple way to confirm that this pub-key would also fail the go-implementation is to feed that old bytes into the internal package `edwards25519`:
```go
var A edwards25519.ExtendedGroupElement
var publicKeyBytes [32]byte
copy(publicKeyBytes[:], publicKey)
asster.False(t, A.FromBytes(&publicKeyBytes))
```
I'll give that a try shortly.

Reply:
OK, so this key won't work in the go implementation either. Tested via:
```go
func TestKey(t *testing.T) {
	publicKey, err := hex.DecodeString("EB6B732C4BD86B5FA3F3BC3DB688DA0ED182A7411F81C2D405506B298FC19E52")
	if err != nil {
		t.Fatalf("%v", err)
	}
	var A edwards25519.ExtendedGroupElement
	var publicKeyBytes [32]byte
	copy(publicKeyBytes[:], publicKey)
	if A.FromBytes(&publicKeyBytes) {
		panic("You shall not pass")
	} else {
	 	t.Log("OK")
	}
}
```

Reply:
What was the take away from this?

Reply:
We should make sure that the test-vectors we use are generated properly, i.e. they don't just satisfy the properties needed for the test-case (here: 32 random bytes did the job) but rather they should be valid domain objects (here: a valid pub-key) - unless of course we want to test that an invalid case.